tools/scoring/tests/dimensions/standard_field_naming_test.py

```diff
@@ -17,7 +17,7 @@
 from absl.testing import absltest
 
 from score.dimensions.standard_field_naming import StandardFieldNaming
-from score.constants import FileTypes
+from score.constants import FileTypes, DimensionCategories
```

Comment:
can you add the constants to the PR? or I am not looking at the right spot?

Reply:
They're at `tools/scoring/score/constants.py`